### ISSUE TITLE
orcid autocomplete, and option

### ISSUE THREAD

**Issue by ValWood** [pombase/canto#1409]
![self select](https://user-images.githubusercontent.com/7359272/38992269-24816562-43d8-11e8-9bb5-e5a458879268.jpg)

When I "self select" a PMID to curate (rather than assigning through the admin interface)   the autocomplete, my orcid ID prefils with e-mail address. Can this be forced to be blank?

Also, perhaps the box should only be available if you add your name/e-mail and the orcid is not already populated...people down want to feel they need to submit it again (or may not remember if they supplied it previously?)

**Comment by ValWood** [comment]
I also notice that some people (presumably who don't have an ORCID) have added some random text!

**Comment by mah11** [comment]
> ... orcid ID prefills with e-mail address. Can this be forced to be blank?

I agree that the ORCID box should never have an email address, but it should only be blank if we don't have an ORCID in Canto for the person.

> Also, perhaps the box should only be available if you add your name/e-mail and the orcid is not already populated

If we already have the ORCID, it should appear in the box (as opposed to not showing the box).

**Comment by ValWood** [comment]
Ideally the box should only appear if  th ORCID is not in the database

**Comment by kimrutherford** [comment]
> Ideally the box should only appear if th ORCID is not in the database

That's tricky because we only know if the ORCID is not in the database once they type in their full email address and there's no reliable way to decide that an email address is complete.

eg. j.smith@something.edu is valid but so is j.smith@something.edu.au

When they're typing an address, at what point should we check the database and show the ORCID box?  Especially when they're typing something less common like some.body@science.unideb.hu 

We could put the ORCID box on the next page after they submit the name and email form but there are already a lot of forms and steps.

**Comment by ValWood** [comment]
I see,  lets discuss. Maybe we can compact  another step. Not urgent...

**Comment by ValWood** [comment]
I'm leaning towards putting the ORCID on the next page. Users will only see this page if they HAVEN'T given their ORCID ID previously (which  can be established once the e-mail is provided). Once it is submitted they will never see this again.

Presenting a blank field  if we already have it recorded
 means that if the user has already  provided but can't remember, and goes to look it up again and then going to look it up and adding it again,  over and over is infinitely more annoying than an extra click for everyone who had not yet added an ORIC.

**Comment by ValWood** [comment]
Does anyone disagree with this solution @mah11 @kimrutherford @Antonialock

**Comment by Antonialock** [comment]
it sounds fine
